Doc/library/idle.rst

```diff
@@ -666,6 +661,42 @@ If ``sys`` is reset by user code, such as with ``importlib.reload(sys)``,
 IDLE's changes are lost and input from the keyboard and output to the screen
 will not work correctly.
 
+User output in Shell
+^^^^^^^^^^^^^^^^^^^^
+
+When a program outputs text, the result is determined by the
+corresponding output device.  When IDLE executes user code, sys.stdout
+and sys.stderr are connected to the display area of IDLE's Shell.  Some of
+its features are inherited from the underlying tcl/tk Text widget.  Others
+are programmed additions.
+
+Text widgets display a subset of Unicode, the Basic Multilingual Plane (BMP).
+Which characters get a proper glyph instead of a replacement box depends on
+the operating system and installed fonts.  Newline characters cause following
+text to appear on a new line, but other control characters are replaced
+with a box.  But note that the ``repr`` function, which is used for
+interactive echo of expression values, replaces control characters
+with escape codes before they are output.
+
+Normal and error output are generally kept separate (on separate lines)
+from code input and each other,  They each get different highlight colors.
+
+For SyntaxError tracebacks, the normal '^' marking where the error was
+detected is replaced by coloring the text with an error highlight.
+When code run from a file causes other exceptions, one may right click
+on a traceback line to jump to the corresponding line in an IDLE editor.
+The file will be opened if necessary.
+
+Shell has a special facility for squeezing output lines down to a
+'Squeezed text' label.  This is done automatically for output over N lines
```

Comment:
Maybe use `:guilabel:`?

Reply:
The guilabel role currently seems useless.  I tried
```
a :guilabel:`Squeezed text` label
```
 and the resulting html markup `a <span class="guilabel">Squeezed text</span> label` has no effect on the rendered page, which simply appears as 'a Squeezed text label', which gives no indication 'text' is a noun modified by 'Squeezed' rather than an adjective modifying 'label'.

This is one of multiple IDLE doc improvements focused on content.  I plan to end with a 'review markup' issue, which I will need help on.